start.bat

```diff
@@ -0,0 +1,3 @@
+start cmd.exe /k "npm run start -w=ui"
```

Comment:
you can also set title of console for better experience
`start "ui" cmd.exe /k "npm run start -w=ui"`
Of course add it to all lines in this file

Reply:
it is not working